frontend/services/screenshot_service/Screenshot_Service.cpp

```diff
@@ -39,6 +39,7 @@ static void log_warning(std::string const& text) {
 // need this to pass GL context to screenshot source. this a hack and needs to be properly designed.
 static megamol::core::MegaMolGraph* megamolgraph_ptr = nullptr;
 static megamol::frontend_resources::GUIState* guistate_resources_ptr = nullptr;
+static megamol::frontend_resources::RuntimeConfig* runtime_config_ptr = nullptr;
```

Comment:
The initial idea was that the RuntimeConfig carries CLI options from the CLI parser to main.cpp. There, each Frontend Service initializes his own Config struct from the RuntimeConfig.

So the `screenshot_show_privacy_note` variable from the RuntimeConfig would havea partner config in the Screenshot_Service Config struct, which is filled in main.cpp and accessed by the Screenshot Service during initialization.

Reply:
Thanks, fixed it in 6acb583 (?)